cmd/query/app/apiv3/http_gateway.go

```diff
@@ -72,11 +72,8 @@ func (h *HTTPGateway) addRoute(
 	if h.TenancyMgr.Enabled {
```

Comment:
Is this needed? Don't we already attach tenancy handler at the server level?

Reply:
we only attach a tenancy interceptors for grpc but I don't see anything for HTTP

Reply:
we need it for http too, but it should be at the server level, not just for apiv3

Reply:
moved the tenancy handler to the server